packages/git/src/browser/diff/git-diff-widget.ts

```diff
@@ -18,19 +16,18 @@ import { LabelProvider } from '@theia/core/lib/browser/label-provider';
 import { GitBaseWidget } from "../git-base-widget";
 import { GitFileChangeNode } from "../git-widget";
 
-@injectable()
```

Comment:
How are widgets different from other classes? I would prefer to keep them injectable, so one can exchange implementations and inject new dependencies.